Cornelica/scenario/007_07.txt

```diff
@@ -42,7 +42,7 @@
 
 @bs f=リリィ@困惑 op=困惑 x=350 y=50 t=200
 
-[Lily] "This place ... I'm not very fond of this place ... so please..."
+[Lily]"This place ...I'm not very fun of this place...\nso please..."
```

Comment:
`I'm not very fun` doesn't make sense. XD Which is why I think "fond" was more suited I believe.

Reply:
yeah... still mindless copy error :D